### PR TITLE
Sort search results

### PR DESCRIPTION
**Pull request by Rasbin**

### CODE REVIEW

File: app/actions/sortActions.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import schemas from 'store/middleware/Schemas';
+import {
+  buildAPIUrl,
+  getErrorTypeDescriptor,
+  getHeadersCreator,
+  getRequestTypeDescriptor,
+  getSuccessTypeDescriptor,
+} from 'utils/apiUtils';
+
+function sortResources(orderBy) {
```
Comment (rBoost) [inline app/actions/sortActions.js]: Why is sort as its own action/api call? It's only a parameter for the search api call.

Comment (Rasbin) [reply app/actions/sortActions.js]: Corrected this in the new commit. Can you verify if it is okay?

File: app/constants/ActionTypes.js [hunk 2]
```diff
@@ -44,6 +44,10 @@ export default {
     SEARCH_RESULTS_GET_REQUEST: 'SEARCH_RESULTS_GET_REQUEST',
     SEARCH_RESULTS_GET_SUCCESS: 'SEARCH_RESULTS_GET_SUCCESS',
 
+    SORT_RESULTS_GET_ERROR: 'SORT_RESULTS_GET_ERROR',
+    SORT_RESULTS_GET_REQUEST: 'SORT_RESULTS_GET_REQUEST',
+    SORT_RESULTS_GET_SUCCESS: 'SORT_RESULTS_GET_SUCCESS',
```
Comment (rBoost) [inline app/constants/ActionTypes.js]: Are these still needed?

Comment (Rasbin) [reply app/constants/ActionTypes.js]: No, missed to remove before. Removed now.

**Pull request merged**